source/reference/kubectl-minio-plugin.rst

```diff
@@ -0,0 +1,102 @@
+:orphan:
+
+.. _minio-kubectl-plugin:
+
+=======================
+MinIO Kubernetes Plugin
+=======================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+Overview
+--------
+
+.. admonition:: Current Stable Version is |operator-version-stable|
+   :class: note
+
+   This reference documentation reflects |operator-version-stable| of the 
+   MinIO Kubernetes Operator and :mc:`kubectl minio` plugin. 
+
+The :mc:`kubectl minio` plugin brings native support for deploying MinIO tenants
+to Kubernetes clusters using the ``kubectl`` CLI. You can use 
```

Comment:
`You can use`  could be replaced with `It can be used`

Reply:
We avoid passive voice when possible.